index.html

```diff
@@ -31,13 +31,17 @@
             <div class="navigation">
                 <a href="">WELCOME</a>
                 <nav class="toggle">
+                    <div class="hamburger">
+                        <button class="close"><i class="fa-solid fa-xmark"></i></button>
+                    </div>
                     <ul class="nav-list">
-                        <li> <a href="#">Portfolio</a> </li>
-                        <li> <a href="#">About</a> </li>
-                        <li> <a href="#">Contact</a> </li>
+                        <li class="link"> <a href="#portfolio">Portfolio</a></li>
+                        <li class="link"> <a href="#About">About</a></li>
+                        <li class="link"> <a href="#contact">Contact</a> </li>
```

Comment:
- [ ] Great work on implementing changes as requested by the previous reviewer, but I can see that you make some on them in half, considering the `gift screen record` he left, the navigation menu should take the full page, neither the user scrolls it, but it is not what happening now, see this image below, onScroll, the nav menu moves together with the page, which is not correct, please fix this and your project will be good to go.
- Giving the `position: fixed;` to the nav menu may solve the problem
- [ ] 
<img width="367" alt="nav" src="https://user-images.githubusercontent.com/79058364/157424418-174a13c0-e8aa-4a88-8ea3-24be86ada225.png">

Reply:
Hi @BenMukebo,
As requested by you, I have:
1. Made the navigation menu to take the full page and,
2.  Made the nav menu stop moving together with the page.